packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -393,6 +393,8 @@ async function fetchLatestSnapshotCore(
                     encodedBlobsSize,
                     sequenceNumber,
                     ops: snapshot.ops?.length ?? 0,
+                    slowTreeStructureCount: snapshot.slowTreeStructureCount ?? 0,
```

Comment:
I'd (in addition to that) add an error event if these numbers are higher than some thresholds. We rarely pay attention to regular events, unless there is a problem. Errors allow us to have simple goal (like no errors, no error count below some number, 99% of sessions to have no errors) that allows to keep an eye on all of our code without need to know about specifics of every feature.